operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1116,7 +1120,8 @@ querier:
   podAnnotations: {}
 
   # Pod Disruption Budget
-  podDisruptionBudget: {}
+  podDisruptionBudget:
+    maxUnavailable: 1
```

Comment:
This will slow down rollouts especially in larger deployments. For the Deployment rollout strategy we use the default of maxUnavailable/maxSurge: 25%. WDYT about having that as the maxUnavailable for the PDB as well?

Reply:
PodDisruptionBudgets don't apply to rollouts, only to actions like draining a node. The rollout strategy on the Deployment / StatefulSet applies to rollouts.

Reply:
oh I see

> Pods which are deleted or unavailable due to a rolling upgrade to an application do count against the disruption budget, but workload resources (such as Deployment and StatefulSet) are not limited by PDBs when doing rolling upgrades. Instead, the handling of failures during application updates is configured in the spec for the specific workload resource.